jekyll/_cci2/credits.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: classic-docs
+title: Using Credits
+categories: [how-to]
+description: How to leverage CircleCI credits
+---
+
+This document describes the basics of using credits with CircleCI. If you are on the legacy CircleCI container-based plan you may want to consider consulting the document on [using containers]({{site.baseurl}}/2.0/containers). If you want to switch from using containers to using credits, please [open a support ticket](https://support.circleci.com/hc/en-us/requests/new) requesting so.
+
+## Overview
+
+The CircleCI credit-based usage plans enable you to only pay for what you use, while also providing flexibility in customizing and scaling your team's CI solution. Credits are used by the minute at varying rates according to what build configuration you use.
```

Comment:
This is close, but not quite accurate.

```
Credits are consumed by the second at varying rates according to what build configuration you use.
```

More specifically, we round every task up to the nearest second, then to the nearest credit. More details here: https://docs.google.com/document/d/1Dt1Jzxz4X5qoIcDL6JjH-zriYxu91pMu5DgNtr59MSk/edit#heading=h.iza1x2ry3j0d